src/clr-angular/data/datagrid/all.spec.ts

```diff
@@ -35,6 +35,7 @@ import DatagridPlaceholderSpecs from './datagrid-placeholder.spec';
 import DatagridRowDetailSpecs from './datagrid-row-detail.spec';
 import DatagridRowSpecs from './datagrid-row.spec';
 import DatagridSpecs from './datagrid.spec';
+import DatagridTabsIntegrationSpecs from './datagrid-tabs.integration.spec';
```

Comment:
This test can be moved into `datagrid.spec.ts`.

Reply:
This test can be moved into `datagrid.spec.ts`.

Reply:
Done, done and done :)